packages/qdrant-loader-core/src/qdrant_loader_core/llm/providers/gemini.py

```
@@ -0,0 +1,350 @@
+                        usage_meta, "candidates_token_count", None
+                    ),
+                    "total_tokens": getattr(usage_meta, "total_token_count", None),
+                }
+
+            return {
+                "text": text,
+                "raw": response,
+                "usage": normalized_usage,
+                "model": getattr(response, "model_version", model_name) or model_name,
+            }
+        except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)